styleguide/templates/styleguide/styleguide.html

```diff
@@ -1,25 +1,52 @@
 {% extends "base.html" %}
 {% load styleguide_tags static %}
 
+{% block head_content %}
+    <link rel=stylesheet href="{% static "styleguide/highlightjs/styles/github.css" %}"  />
+    <link rel=stylesheet href="{% static "styleguide/styleguide.css" %}" />
+{% endblock %}
+
```

Comment:
If we're going to override more blocks we expect to have available from the project's own base.html, we should use block names that are more common, especially with our own projects. Look at the base.html from our project template: https://github.com/caktus/django-project-template/blob/master/project_name/templates/base.html